yml/ivoa/obscore_nominal.yaml

```diff
@@ -538,7 +656,8 @@ tables:
     votable:utype: Char.ObservableAxis.Accuracy.StatError.Refval.value
     tap:std: 1
     tap:principal: 0
-    ivoa:unit: same units as the value of the o_unit attribute
+    ivoa:unit:
```

Comment:
@stvoutsin / @JeremyMcCormick it's this change that fixes the validation error. "same units as..." is not a valid unit.

Reply:
I understand exactly why this **would** fail a validation, but is this theoretical, or is it **right now** failing validation in some place we can't see?

Reply:
It wasn't theoretical in the sense that I couldn't use this yaml file in dax_obscore without everything breaking.

Reply:
(Granted I can't remember the details now because it was a while back)

Reply:
OK, that explains why we can't see an actual failure in logs.

Reply:
Do you attempt to use this actual file, not a copy of it, programmatically, from dax_obscore?

Reply:
At the moment there is a copy in dax_obscore. Once this is merged and combined with the recent change to use python package resources I will be able to delete the local copy and use this one.